app/delivery/send_to_providers.py

```diff
@@ -253,13 +252,10 @@ def get_logo_url(
     return 'https://{}.{}/{}'.format(bucket, domain, logo_file)
 
 
-def get_html_email_options(
-    notification,
-    provider,
-):
+def get_html_email_options(notification: Notification) -> Dict[str, str | bool]:
```

Comment:
Please use Optional/Union, not |.

Reply:
https://github.com/department-of-veterans-affairs/notification-api/pull/1860#discussion_r1681690527

Reply:
This one's on me, I recommended it because python 3.10 supports it and I use it in the QA Suite.

Reply:
Updated.